docs/resources/aws_s3_bucket.md

```diff
@@ -0,0 +1,123 @@
+---
+title: About the aws_s3_bucket Resource
+---
+
+# aws_s3_bucket
+
+Use the `aws_s3_bucket` InSpec audit resource to test properties of a single AWS bucket.
+
+To test properties of a multiple S3 buckets , use the `aws_s3_buckets` resource.
+
+<br>
+
+## Limitations
+
+S3 bucket security is a complex matter.  For details on how AWS evaluates requests for access, please see [the AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/how-s3-evaluates-access-control.html).  S3 buckets and the objects they contain support three different types of access control: bucket ACLs, bucket policies, and object ACLs.
+
+As of this writing, this resource only supports evaluating the first two, bucket ACLs and bucket policies.  Evaluating object ACLs introduces scalability concerns that are currently not well handled by the AWS API, and may be better handled via AWS mechanisms such as CloudTrail and Config.
```

Comment:
As of January 2018, this resource supports evaluating bucket ACLs and bucket policies.  We do not support evaluating object ACLs because it introduces scalability concerns in the AWS API; we recommend using AWS mechanisms such as CloudTrail and Config to evaluate object ACLs.